scripts/apps/search/components/fields/state.tsx

```diff
@@ -26,7 +26,7 @@ export const state: React.StatelessComponent<any> = (props) => {
                 className: 'state-label state-' + props.item.state,
                 key: 'state',
             },
-            $filter('removeLodash')(gettextCatalog.getString(props.item.state)),
+            $filter('removeLodash')(gettext(props.item.state)),
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.